src/tag/tag-filter.component.ts

```diff
@@ -20,7 +21,17 @@ import { Tag } from "./tag.component";
 	`
 })
 export class TagFilter extends Tag {
+	/**
+	 * Function for close/delete the tag
+	 *
+	 */
+	@Output() close: EventEmitter<any> = new EventEmitter<any>();
+
 	@HostBinding("attr.class") get attrClass() {
 		return `bx--tag bx--tag--filter bx--tag--${this.type} ${this.class}`;
 	}
+
+	closeFn() {
```

Comment:
Just a FYI, generally we [follow these naming conventions](https://github.com/IBM/carbon-components-angular/wiki/Style-guide#event-handler-and-emitter-handler-naming)

So this should be 
```suggestion
	doClose() {
```

_however_ since we don't need to do any extra processing, you can just call `close.emit()` in the template 👍